src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -9,8 +11,17 @@ interface ChannelClaimTableRowProps {
   claimedChannel: ClaimedChannel;
 }
 const StyledTableCell = styled(TableCell)({
-  verticalAllign: 'top',
+  verticalAlign: 'top',
 });
+
+const StyledChip = styled(Chip)({
+  '.MuiChip-label': {
+    whiteSpace: 'nowrap',
+  },
+  variant: 'filled',
+  color: 'secondary',
```

Comment:
`styled` tar bare css-in-js, så props kan ikke sendes her. Så disse har vel ingen effekt? De må settes som vanlige props i så fall
```suggestion
```

Reply:
ellers er et enkelt triks å kommentere inn og ut, da ser man at det popper mellom de ulike versjonene